cpp/dependency_scan/metadata/README.md

```
@@ -0,0 +1,758 @@
+
+Each `.yml` file represents one dependency:
+
+```
+metadata/
+├── _template.yml       # Template for new dependencies
+├── _schema.yml         # JSON schema for validation
+├── dpkg.yml           # System libraries (list format)
+├── tensorrt-llm.yml   # Individual dependency
+├── pytorch.yml
+├── cuda-cudart-12.yml
+└── ... (62+ dependencies)
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)